crates/bevy-inspector-egui/src/quick.rs

```diff
@@ -467,33 +468,36 @@ impl System for BoxedConditionHelper {
     fn is_exclusive(&self) -> bool {
         self.0.is_exclusive()
     }
-
-    unsafe fn run_unsafe(&mut self, input: Self::In, world: &World) -> Self::Out {
+    unsafe fn run_unsafe(
+        &mut self,
+        input: Self::In,
+        world: bevy_ecs::world::unsafe_world_cell::UnsafeWorldCell,
+    ) -> Self::Out {
         // SAFETY: same as this method
         unsafe { self.0.run_unsafe(input, world) }
     }
 
-    fn apply_buffers(&mut self, world: &mut World) {
-        self.0.apply_buffers(world)
+    fn apply_deferred(&mut self, world: &mut World) {
+        self.0.apply_deferred(world)
     }
 
     fn initialize(&mut self, _world: &mut World) {
         self.0.initialize(_world)
     }
 
-    fn update_archetype_component_access(&mut self, world: &World) {
+    fn update_archetype_component_access(
+        &mut self,
+        world: bevy_ecs::world::unsafe_world_cell::UnsafeWorldCell,
```

Comment:
I imported `UnsafeWorldCell` and `Tick` at the top here for marginally better readability